tracer/build/_build/docker/universal.dockerfile

```diff
@@ -0,0 +1,94 @@
+﻿FROM datadog/libddwaf:toolchain as base
```

Comment:
`libddwaf:toolchain` uses `debian:12.1` as a base: https://github.com/DataDog/libddwaf/blob/445c1a8f2471c0461aeace0ae868924c8af01133/docker/libddwaf/toolchain/Dockerfile#L1

_Currently_ we build against debian 10 for glibc compat. Am I right in assuming that doesn't matter here, because of how we're loading as the universal binary? ~~We _should_ still work on debian 10 right?~~ Our tests all use debian 10, so yeah, I'm sure we're good 👍

Reply:
Exactly